<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogA</class>
 <widget class="QDialog" name="dialogA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="AVION">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>541</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_identifiant">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_capacite">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_Ajouter">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(0, 255, 0);
</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>247</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo 1.3.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>481</x>
       <y>213</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo4.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>-180</x>
       <y>-20</y>
       <width>1111</width>
       <height>361</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image8.jpg</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_etat">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Fonctionnel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Endommage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En reparration</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_type">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>jet prive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>avion de tourisme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>avion de transport</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>helicoptere</string>
      </property>
     </item>
    </widget>
    <zorder>label_16</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>le_identifiant</zorder>
    <zorder>le_capacite</zorder>
    <zorder>pb_Ajouter</zorder>
    <zorder>label_14</zorder>
    <zorder>label_15</zorder>
    <zorder>cb_etat</zorder>
    <zorder>cb_type</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id_m">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>6</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_capacite_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
color: rgb(0, 0, 0);
</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_modifier">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_etat_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Fonctionnel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Endommage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En reparration</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_type_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>jet prive</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>avion de tourisme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>avion de transport</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>helicoptere</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>-210</x>
       <y>-50</y>
       <width>881</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image9.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>lineEdit_id_m</zorder>
    <zorder>le_capacite_2</zorder>
    <zorder>pushButton_modifier_2</zorder>
    <zorder>cb_modifier</zorder>
    <zorder>cb_etat_2</zorder>
    <zorder>cb_type_2</zorder>
   </widget>
   <widget class="QWidget" name="Afficher">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QPushButton" name="pb_Afficher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_imprimer">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_trier">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>240</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Capacité</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Etat</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_docExel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Document Excel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo4.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>581</width>
       <height>351</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image6.jpg</pixmap>
     </property>
    </widget>
    <widget class="QTableView" name="tab_afficher">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 200, 255);
border-top-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 255), stop:0.33 rgba(0, 0, 0, 255), stop:0.34 rgba(255, 30, 30, 255), stop:0.66 rgba(255, 0, 0, 255), stop:0.67 rgba(255, 255, 0, 255), stop:1 rgba(255, 255, 0, 255));</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_trier">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>240</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_rechercher">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>270</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_rechercher">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>pb_Afficher</zorder>
    <zorder>pushButton_imprimer</zorder>
    <zorder>comboBox_trier</zorder>
    <zorder>pushButton_docExel</zorder>
    <zorder>label_18</zorder>
    <zorder>tab_afficher</zorder>
    <zorder>pb_trier</zorder>
    <zorder>pb_rechercher</zorder>
    <zorder>cb_rechercher</zorder>
   </widget>
   <widget class="QWidget" name="Supprimer">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 127);</string>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 30, 30);

</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>90</y>
       <width>261</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/logo4.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-8</y>
       <width>541</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/Esprit/2eme année/semestre 1/projet cpp/logo/image3.jpg</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_suppression">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>21</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-bottom-color: rgb(255, 255, 0);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_suppression">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tb_supprimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>521</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <zorder>label_19</zorder>
    <zorder>label_12</zorder>
    <zorder>label_5</zorder>
    <zorder>pb_supprimer</zorder>
    <zorder>cb_suppression</zorder>
    <zorder>le_suppression</zorder>
    <zorder>tb_supprimer</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
